spi/src/test/resources/serialized_asset.json

```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
```

Comment:
Wouldn't be neater to remove this level of nesting? Just serializing the properties at the root level.